src/Microsoft.Diagnostics.Runtime/ClrHeap.cs

```diff
@@ -19,13 +19,18 @@ namespace Microsoft.Diagnostics.Runtime
     public abstract class ClrHeap
     {
         /// <summary>
+        /// Returns a ClrType from an EE address. Available in .NET native only. Used for constructing ClrType object from an EEType value encoded in a .NET native serialized exception.
+        /// </summary>
+        public abstract ClrType GetClrTypeFromEE(ulong eeType);
```

Comment:
You should not create a function for this.  Instead override GetTypeByMethodTable and provide your implementation there.